### ISSUE TITLE
check log for error

### ISSUE THREAD

**Issue by nammminn** [DxCx/plugin.video.9anime#355]
I got a check log for error and this shows in the log 
https://i.imgur.com/nZ4LKhl.png

**Comment by Apex90** [comment]
yea noticed this when the search function no longer works. Well its been a fun run guys, looks like this is the end of the road for this addon now that DxCx is on to making another addon and has deprecated this one.

**Comment by SwagOtaku** [comment]
Just need to tweak the url to get it working again.

Just go into NineAnimeBrowser.py and change ''' _BASE_URL = "https://9anime.is" '''
to ''' _BASE_URL = "https://9anime.to" '''

**Comment by JulGer** [comment]
Wow. Can't close the popups fast enough.

**Comment by d1arkstar** [comment]
@SwagOtaku  thanks and also for stating what file to change it in

**Comment by Aguzy** [comment]
Thank you SwagOtaku, works like a charm

**Comment by Thekozmicpig** [comment]
sprink17, go to add ons, install from zip, external storage, downloader and it should be in there.

**Comment by sprink17** [comment]
That’s what I thought, but all I see is old zips. Nothing with the master.zip :(. I shall try again. I doo appreciate everybody’s help tho

**Comment by sprink17** [comment]
In downloader, I’m just copying https://www46.zippyshare.com/v/5AEv51qq/file.html right or am I dumb. Sorry I’m pretty ignorant to this stuff

**Comment by Aguzy** [comment]
No you only have to put in the https://bit.ly/ the link has been shortened, try https://bit.ly/2O9gSGl url in downloader

**Comment by sprink17** [comment]
I’m a moron. And all you guys are the best

**Comment by sprink17** [comment]
Ughh man I’m sorry, just don’t know what’s going on. I got it to show up in Kodi with that most recent Bit.ly link you gave me, but Kodi is saying it’s unable to install from zip file.
The contents of the file seem in order on downloader so it should be correct. What could I have screwed up?

**Comment by sprink17** [comment]
Actually in downloader if I click on it it says “bad zip”. That’s after downloading the https://bit.ly/2O9gSGl

**Comment by Aguzy** [comment]
I just done it again through downloader and it worked with the https://bit.ly/2O9gSGl not sure what is going wrong with it, give it another try please and if it doesn't work I'll make a new one for you

**Comment by sprink17** [comment]
https://imgur.com/a/0I2k7ek

**Comment by Aguzy** [comment]
https://bit.ly/2zUxhWq try that one sprink17, I tested it just there and its working spot on, let me know how you get on

**Comment by sprink17** [comment]
Alright will do.
Just don’t get why I can get the file downloaded but not install it :( 
I’m wondering if because I didn’t go change the URL manually like Swag mentioned above there’s some disconnect. Idk, I can’t act like I know much about all this

**Comment by Aguzy** [comment]
No it still works from just the zip, when I installed it there I done it onto a clean Kodi fork that's never had the add-on on it to make sure it was working ok, only thing I can think it was something to do with my Dropbox link or bit.ly, there's a different one with a tiny URL https://tinyurl.com/yd65nv49 , hope you get it going bud

**Comment by sprink17** [comment]
Alright downloaded it in downloader. Appears to be A duplicate of the previous file so I’m downloading the correct source.  Went over to Kodi Addons,
Install from zip/external storage/download and tried installing from that zip file and it gave me the same message I got in the video I linked above claiming it can’t install from zip. 
After you downloaded the file in downloader you just gave me, when you click “unzip” (In downloader) does it give you a “bad zip” Message. I know you don’t need to unzip it but that’s how I’ve double checked zips in the past.

**Comment by sprink17** [comment]
Okay I gotcha. Thanks for the help sincerely. 
I may try to reinstall 9anime entirely, idk :(

**Comment by sprink17** [comment]
Yeah no luck with the tiny URL. It’s downloading fine, just claims the zip is bad. Have no idea man. Thanks again tho. I’ll try to find another add on I suppose

**Comment by clutchylfg** [comment]
Thanks fam!!
